microsoft-edge/devtools-guide-chromium/3D-view.md

```diff
@@ -0,0 +1,77 @@
+---
+description: All about 3D View and how to use it.
+title: 3D View
+author: erdraud
+ms.author: erdraud
+ms.date: 11/27/2019
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+
+# 3D View
+
+Use the **3D View** to debug your web application by navigating through the Document Object Model (DOM) or the z-index stacking context. With it you can: 
+
+- [Explore the web page translated into a 3D perspevtive](#3D-DOM)
+- [Debugging based on z-index stacking context](#Z-index)
+- [Clear some of the clutter in the DOM pane](#Changing-your-view) or the [z-index pane](#Change-the-scope-of-your-exploration)
+- [Pick the color scheme to best debug your DOM problems](#DOM-color-type) or [z-index problems](#Z-index-color-type)
+
+There are two panes that you can use for your debugging experience.
+
+1. **Z-index** Navigate through the different elements in the web application with the z-index context in mind. This is the default pane.
+2. **3D DOM** Explore the DOM as a whole with all the elements at your fingertips. To access this pane, click on the "DOM" pane next to the "Z-index" pane.
+
+![Z-index view](./media/z-index-view.png)
+
+## Z-index
+
+While the Z-index pane has shared features with the DOM pane, they still have elements that are unique to the pane.
+
+### Highlight elements with stacking context
+
+This setting allows you to toggle the z-index tags on and off for the elements in the canvas. The checkbox will be selected by default.
+
+### Change the scope of your exploration
+
+The **Show all elements** button is the quickest way to display all the DOM's elements after changing the settings below.
+
+**Show only elements with stacking context** removes elements without stacking context and flattens the DOM for easier navigation.
+
+The **Isolate selected element** filter is essentially three buttons in one. There are two checkboxes below this button: one is "Show all parents" and "Keep only parents with new stacking context." 
+
+The "Show all parents" checkbox will be checked by default. If you select an element in the canvas pane and click on **Isolate selected element**, the canvas will only display the element and its parents.
+
+If you select the "Keep only parents with new stacking context," and click on **Isolate selected element**, the canvas will only display the element and the parents that have a new stacking context. **EXPLAIN NEW STACKING CONTEXT**
```

Comment:
I think this is a todo left in the Markdown. Are you planning on completing this todo or should we just remove it?